### PR TITLE
add jsonnetfmt hook and sort repos

### PR DESCRIPTION
**Pull request by gaurav517**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 /build
 /node_modules
 /venv
+.idea
```
Comment (asottile) [inline .gitignore:9]: please don't touch the .gitignore on projects you don't own

File: all-repos.yaml [hunk 2]
```diff
@@ -1,173 +1,174 @@
...
-- https://github.com/pre-commit/mirrors-puppet-lint
-- https://github.com/pre-commit/mirrors-ruby-lint
-- https://github.com/pre-commit/mirrors-scss-lint
-- https://github.com/pre-commit/mirrors-yapf
-- https://github.com/pre-commit/pygrep-hooks
-- https://github.com/FalconSocial/pre-commit-mirrors-pep257
-- https://github.com/FalconSocial/pre-commit-python-sorter
-- https://github.com/guykisel/pre-commit-robotframework-tidy
-- https://github.com/guykisel/prospector-mirror
+# - https://github.com/hakancelik96/unimport
```
Comment (asottile) [inline all-repos.yaml:2]: this file is intentionally not sorted, please append your repository to the bottom

Comment (gaurav517) [reply all-repos.yaml:2]: 👍  will append to end. Out of curiosity: why not sorted? (could help in duplicate detection?)

Comment (asottile) [reply all-repos.yaml:2]: it populates https://pre-commit.com/hooks.html

Comment (gaurav517) [reply all-repos.yaml]: hmm then won't the page be alphabetically populated?

Comment (asottile) [reply all-repos.yaml]: pre-commit doesn't start with an "a" does it?

**Review by asottile — APPROVED** [review]

**Pull request merged**